SA3/qa_web/models.py

```diff
@@ -46,12 +52,22 @@ class Comments(Post):
     """
     question = models.ForeignKey(Questions, null=True, on_delete=models.CASCADE)
     answer = models.ForeignKey(Answers, null=True, on_delete=models.CASCADE)
-    
+    voters = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='c_voters', through='Vote')
+
 class Tags(models.Model):
     """
     One or multiple tags can be assigned to a question, in order to classify the question
     There can be many tags to a question
     """
     question = models.ForeignKey(Questions, null= True, on_delete=models.CASCADE)
     category = models.CharField(max_length=50, null=True)
-    
\ No newline at end of file
+
+class Vote(models.Model):
+    """
+    Defines the relationship between a user voting and the post
+    """
```

Comment:
I read up on many-to-many relationships in Django and this seems to be the correct implementation for an intermediate class that handles relationships with foreign keys and the fields that call it with the `through` parameter.